db.py

```
@@ -0,0 +1,12 @@
+# db.py
+from sqlalchemy import create_engine, Column, Integer, String, Float, Date, Boolean
```

Ruff (0.12.2)

`sqlalchemy.Column` imported but unused

Remove unused import

(F401)